big_tests/tests/sasl_external_SUITE.erl

```diff
@@ -420,7 +421,8 @@ generate_user(C, User, Transport) ->
     UserCert = maps:get(User, Certs),
 
     Common = [{username, list_to_binary(User)},
-	      {server, <<"localhost">>},
+	      {server, domain()},
```

Comment:
Could you realign these lines to make all the opening `{` have the same indentation as `{username` ?